src/_pytest/pytester.py

```diff
@@ -1032,7 +1034,7 @@ def collect_by_name(self, modcol, name):
             if colitem.name == name:
                 return colitem
 
-    def popen(self, cmdargs, stdout, stderr, **kw):
+    def popen(self, cmdargs, stdout, stderr, stdin=CLOSE_STDIN, **kw):
```

Comment:
Also making `stdout` and `stderr` have a default here would be nice.  What do you think?
They could just default `subprocess.PIPE`.

Reply:
Done.